agiverse/agent/messaging.py

```diff
@@ -255,8 +262,8 @@ async def _generate_model_response(self, websocket):
             inventory=format_json(self.inventory_data),
             state=format_json(self.state_data),
             available_actions=format_json(self.available_actions),
-            working_memory=format_memory(self.agent.working_memory),
-            long_term_memory=self.agent.long_term_memory,
+            working_memory=self.agent.working_memory.steps_to_string(),
```

Comment:
let's remove agent.working_memory and agent.long_term_memory if they are no longer used